evalml/automl/automl_search.py

```diff
@@ -796,30 +661,32 @@ def __init__(
             self.X_train.ww.schema,
             self.y_train.ww.schema,
         )
-        self.allowed_model_families = list(
-            set([p.model_family for p in (self.allowed_pipelines)])
-        )
 
-        self.logger.debug(
-            f"allowed_pipelines set to {[pipeline.name for pipeline in self.allowed_pipelines]}"
-        )
-        self.logger.debug(
-            f"allowed_model_families set to {self.allowed_model_families}"
+        text_in_ensembling = (
```

Comment:
Opted to leave this out as both `IterativeAlgorithm` and `DefaultAlgorithm` take in `text_in_ensembling` as arguments.